### PR TITLE
added supplier policy

### PR DESCRIPTION
**Pull request by andreas-kater**

### CODE REVIEW

File: handbook/information-security/supplier-policy.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+## Vetting Process
+The below diagram illustrates the process that the rest of this document
+describes in more detail.
+
+![Supplier Policy Flowchart](../../assets/supplier-policy.png)
+
+### Supplier List
+
+Our [Supplier
+List](https://docs.google.com/spreadsheets/d/14kog_nFut3v2t5mZNNatZ-oWb7tP3ImWdXjUP9EkZGc/edit?usp=sharing)
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md]: Not sure I would have this link here. It's not publicly available (or shouldn't be) and we can just reference that the doc exists.

File: handbook/information-security/supplier-policy.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+tracks the suppliers we interact with.
+
+This is checked for completeness monthly by the [Information Security
+Officer](../README.md#contacts).
+
+We define suppliers as any organisation that has invoiced us. Therefore the
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md]: I think we need to change this. If we just started working with someone, they may not have invoiced us yet. They may also be providing services in return for our services.

File: handbook/information-security/supplier-policy.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+monthly update requires a review of our received invoices for new suppliers.
+
+### Supplier Ownership
+
+The Information Security Officer will assign a Supplier Owner to each Supplier
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md]: Link to the ISO?

File: handbook/information-security/supplier-policy.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+in the Supplier List. These owners are tracked in the Supplier List. They are
+responsible for conducting the Vetting Process laid out below.
+
+### Vetting Process
+
+#### Vetting Requirement
+
+While we work with a large number of suppliers, not all of them will have access
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md]: This is a bit wordy. I would probably say we organise suppliers into 3 categories. Define the categories along with the associated checks/measures against each supplier.

File: handbook/information-security/supplier-policy.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+In order to assess an individual supplier, the Supplier Owner will review the
+publicly available information of the supplier's information security policies
+and processes.
+
+If there is not sufficient information available publicly to conduct the review,
+the Supplier Owner will send out the [Supplier Security
+Questionnaire](https://goo.gl/forms/3J0BmcrOIhUxGP2x1) to the supplier to gather
+further information. Responses to the Questionnaire can be found
+[here](https://docs.google.com/spreadsheets/d/1q4TB_HkJubEPoc4MjeColk0LG9AUr-6_2d_UvrnbY4g/edit?usp=sharing).
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:75]: Is this public?

File: handbook/information-security/supplier-policy.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+#### Rejecting a Supplier
+
+If a supplier has been rejected the commercial relationship should be ended
+within two months of the decision to reject.
+
+#### Accepting a Supplier
+
+If a supplier has been accepted the commercial relationship can continue and a
+review of the supplier's policies should be scheduled. This is tracked in the [Supplier
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md]: Again, wouldn't make this public if it is.

**Review by tom-mullen — CHANGES_REQUESTED** [review]

File: handbook/information-security/supplier-policy.md (hunk 2 above)
Comment (andreas-kater) [reply handbook/information-security/supplier-policy.md]: How about "We define suppliers as any organisation that we collaborate with in providing our clients with our services"?

File: handbook/information-security/supplier-policy.md (hunk 5 above)
Comment (andreas-kater) [reply handbook/information-security/supplier-policy.md:75]: No, my approach was to put links in there for the internal audience and if someone external tries to access the link they are asked to request permission. 

That way our team know where to reference it. Agreed that it's not ideal if someone external tries to access the link, but hopefully they'd understand that not all information can be public. At least they would know how we track things. 

On balance I think it's better to have the links.

File: handbook/information-security/supplier-policy.md (hunk 4 above)
Comment (andreas-kater) [reply handbook/information-security/supplier-policy.md]: Would the three categories be:
- Large organisation -> review of their publicly available info sec policies
- Medium-sized organisation -> review of their publicly available info sec policies + questionnaire where necessary + negotiation where necessary?
- Individuals and small organisations -> NDA + negotiation?

File: handbook/information-security/supplier-policy.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
+# Supplier Policy
+
+In providing our services we rely on a number of third party suppliers that have
+access to some of the sensitive information we handle. This policy sets out how
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:4]: I think we should specify which category of information they have access to.

File: handbook/information-security/supplier-policy.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+* ensure the relevant suppliers follow best practices to minimise information
+  security risks
+* work with our suppliers to implement best practice where it is not being
+  implemented
+* make a conscious decision whether the risks we expose ourselves to are
+  acceptable
+* where we are exposed to unacceptable risk, mitigate this by ending the
+  collaboration if necessary
+
+## Vetting Process The below diagram illustrates the process that the rest of
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:22]: Looks like a typo.

File: handbook/information-security/supplier-policy.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+this document describes in more detail.
+
+![Supplier Policy Flowchart](../../assets/supplier-policy.png)
+
+### Supplier List
+
+We define suppliers as any organisation that we collaborate with in providing
+our clients with our services.
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:30]: ...services to our clients.

File: handbook/information-security/supplier-policy.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+
+Our Supplier List tracks the suppliers we interact with. This is checked for
+completeness monthly by the [Information Security
+Officer](../README.md#contacts). As part of this check, we make sure all
+entities that have ever invoiced us are on the Supplier List.
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:35]: Do we maintain all supplier historically? Sounds cumbersome. It's also contradictory to your previous definition of who this policy applies to.

File: handbook/information-security/supplier-policy.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+Supplier List. They are responsible for conducting the Vetting Process laid out
+below.
+
+### Vetting Process
+
+#### Vetting Requirement
+
+We organise suppliers into four categories:
+
+* Organisations without access to sensitive information.
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:50]: Categorise data in line with our process.

File: handbook/information-security/supplier-policy.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+  * These do not require a security vetting.
+* Individuals and small organisations
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:52]: What is a small organisation?

File: handbook/information-security/supplier-policy.md [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+  * These are likely to require an NDA to be signed and/or a negotiation around
+    how to address vulnerabilities
+* Medium-sized organisation
+  * These are likely to require a review of their publicly available info sec
+    policies, a questionnaire where necessary and negotiations to address
+    vulnerabilities where necessary
+* Large organisation
+  * These are likely to require a review of their publicly available info sec
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:60]: info sec -> Information Security

File: handbook/information-security/supplier-policy.md [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+    policies. Given they are significantly larger than us, it is usually not
+    feasible to require a questionnaire and/or negotiations. Therefore a
+    decision to collaborate or not will be made based solely on the publicly
+    available information.
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:64]: There is a full-stop at the end of this sentence, whereas there is not at the end of all others.

File: handbook/information-security/supplier-policy.md [hunk 15]
```diff
@@ -0,0 +1,119 @@
...
+
+The Supplier Owner decides whether supplier vetting is required for an
+individual  supplier.
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:67]: extra space

File: handbook/information-security/supplier-policy.md [hunk 16]
```diff
@@ -0,0 +1,119 @@
...
+
+If a supplier vetting is required, this is to be completed within two months of
+the supplier being added to the Supplier List.
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:70]: What do we do for historical suppliers?

File: handbook/information-security/supplier-policy.md [hunk 17]
```diff
@@ -0,0 +1,119 @@
...
+
+#### Vetting Assessment
+
+In conducting the Vetting Assessment, the Supplier Owner assesses the available
+information and answers the following questions:
+
+1. Do we as an organisation sufficiently understand the risks of working with
+this supplier?
+
+2. Has the supplier adequately mitigated the risk of security incidents to
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:88]: How do we define whether this is the case or not?

File: handbook/information-security/supplier-policy.md [hunk 18]
```diff
@@ -0,0 +1,119 @@
...
+minimise the probability of them occurring?
+
+3. Does the supplier's incident reporting policy allow us to effectively manage
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:91]: How do we decide this?

File: handbook/information-security/supplier-policy.md [hunk 19]
```diff
@@ -0,0 +1,119 @@
...
+If the answer to all three questions is yes, the Supplier Owner accepts the
+supplier.
+
+#### Negotiating Mitigations
+
+If the answer to any of the three Vetting Assessment questions is no, the
+Supplier Owner may negotiate with the supplier to try to better mitigate the
+risks.
+
+Where such negotiations are entered, the final decision may be delayed by up to
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:103]: Negotiation sounds like the wrong word. We shouldn't be negotiating on risk. We should assess it and set clear milestones for progress along with a timeline to implementation.

If the milestones aren't reached and the risk is outside of our tolerance, then the supplier needs to be rejected.

File: handbook/information-security/supplier-policy.md [hunk 20]
```diff
@@ -0,0 +1,119 @@
...
+
+#### Rejecting a Supplier
+
+If a supplier has been rejected the commercial relationship should be ended
+within two months of the decision to reject.
+
+#### Accepting a Supplier
+
+If a supplier has been accepted the commercial relationship can continue and a
+review of the supplier's policies should be scheduled. This is tracked in the
```
Comment (tom-mullen) [inline handbook/information-security/supplier-policy.md:116]: How often?

**Review by tom-mullen — CHANGES_REQUESTED** [review]

File: handbook/information-security/supplier-policy.md (hunk 2 above)
Comment (tom-mullen) [reply handbook/information-security/supplier-policy.md]: We define suppliers as any organisation that we collaborate with in providing services to our clients.

**Pull request closed without merging**